ui/admin/app/routes/scopes/scope/credential-stores/index.js

```diff
@@ -81,7 +81,7 @@ export default class ScopesScopeCredentialStoresIndexRoute extends Route {
     if (totalItems > 0) {
       return true;
     }
-    const options = { pushToStore: false };
+    const options = { pushToStore: false, peekIndexedDB: true };
```

Comment:
Perhaps im missing something but I'm thinking that wouldn't cause any issues because the indexedDB will still be populated with the data under the specified scope.

Reply:
Oh yeah good point, the request still grabs all the data before we filter, nvm good call ignore this then